### ISSUE TITLE
iOS Bootcamp Demo App Submission - Tip Calculator

### ISSUE THREAD

**Issue by stepchud** [stepchud/Just-the-Tip#1]
This is my solution for the demo app submission. See README.md for details.
/cc @codepathreview

**Comment by codepathreview** [comment]
Thanks for submitting your pre-work but your current submission is incomplete. Can you please review the section in the pre-work about [the readme]https://courses.codepath.com/snippets/intro_to_ios/prework.md#heading-5-submitting-the-project) for your project? 

In particular, be sure to start from the README template that is linked there, check off your completed required and optional features (`[x]`), list any additional features, and [include a recorded app walkthrough with LiceCap](http://www.cockos.com/licecap/). 

Please re-submit your project on your [application dashboard](https://apply.codepath.com/dashboard/), and it will notify us to review again.

**Comment by stepchud** [comment]
@codepathreview Thanks for the update. I originally submitted before the application process was updated. I noticed a few of the things have changed so I will update my application to reflect those changes. I also am running the latest version of Xcode (8.0) with iOS 10 and Swift 3. I had to make several changes to the project to fix those issues, e.g. NSUserDefaults renamed to UserDefaults etc., so I wanted to make sure this is acceptable.
